src/pages/CompanyEditPage.tsx

```diff
@@ -53,7 +53,13 @@ export function CompanyEditPage() {
     );
   }
 
-  if (error) {
+  if (
+    error &&
+    !(
+      typeof (error as any).status === "number" &&
+      ((error as any).status === 401 || (error as any).status === 403)
+    )
```

Comment:
This logic would be nice to clean up. Maybe use some temporary variables to clarify what we are looking for?